admin/src/App.tsx

```diff
@@ -1,12 +1,15 @@
-import React from "react";
 import "./App.css";
+import Layout from "./Components/Layout/Layout"
+import { BrowserRouter as Router } from "react-router-dom";
 
 
 function App() {
   return (
-    <div className="App">
-      Admin App should be here.
-    </div>
+    <Router>
+      <div className="App">
+        <Layout />
+      </div>
```

Comment:
Try putting the Routes here and only an Outlet in the Layout.
Also, please put the Router in the index.ts (wrap the whole App component) otherwise we will not be able to use react-router-dom hooks inside the app component.

Reply:
https://stackblitz.com/edit/github-6zktaw?file=src/App.tsx